src/css/skins/seven.less

```diff
@@ -129,55 +121,63 @@
     /* Styles for play button on idle */
     .jw-display-icon-container {
         border-radius: 50%;
-        border: 1px solid #333;
+        width: 4.5em;
+        height: 4.5em;
+        margin: -2.25em auto 0;
+        
```

Comment:
The icon size and centering should also be improved:

<img width="138" alt="screen shot 2016-06-03 at 4 37 34 pm" src="https://cloud.githubusercontent.com/assets/333258/15792235/edf2215c-29a9-11e6-9249-437d793fd68f.png">